keymaker/worker.py

```diff
@@ -2,7 +2,7 @@
 
 parser = argparse.ArgumentParser(
     description='Worker file for repo statistics.')
-parser.add_argument('-a', '--watchlist', type=str, default=None, required=True,
+parser.add_argument('-a', '--watchlist', type=str, default='watchlist.txt', required=False,
```

Comment:
Won't we always have repos in `watchlist.txt`?

I'd imagine we only need to specify this argument if we want to parse a file that's not in the default (ex. `./new_location/new_watchlist.txt`) 
Right? 

Otherwise, we can just assume that `./watchlist.txt` is where all the repos are specified.

Reply:
Oh I see...
ok yeah, that's fine; didn't realize the default was set to `watchlist.txt`